icu4j/main/translit/src/main/java/com/ibm/icu/text/BreakTransliterator.java

```diff
@@ -52,8 +50,9 @@ public void setInsertion(String insertion) {
     public BreakIterator getBreakIterator() {
         // Defer initialization of BreakIterator because it is slow,
         // typically over 2000 ms.
-        if (bi == null) bi = BreakIterator.getWordInstance(new ULocale("th_TH"));
-        return bi;
+        // Using a holder class for safe init without a volatile-read.
+        if (bi == null) bi = WordBreakIteratorHolder.BI;
+        return (BreakIterator) bi.clone();
```

Comment:
The Break transliterator is a bit different - since the `bi` is shared, we now have to clone it for each operation to get rid of `synchronized`. This assumes that `clone()` is reasonably efficient.

Reply:
The high bit is that it should be reasonably efficient.

I poked around, and It does have some state. But I think the only things that require much processing are some int[] buffers, either directly or via fBreakCache and fDictionaryCache.

The biggest chunk of data is in fData. But that is immutable, and so is just copied; there is no need for a deep clone.

Reply:
Thanks for confirming. The benchmark also indicates that it's faster to clone a new break iterator, than it is to synchronize on a shared one, with multiple cores involved.